utils/extract-strings.js

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env ./node_modules/.bin/babel-node
+// @flow
+/**
+ * Extracts i18n strings from each package and writes them out to <package>/dist/strings.js.
+ *
+ * This file contains i18n._(), i18n._$(), and i18n.ngettext() calls that can be re-extracted
+ * by webapp.  This is required for so that we can leverage webapp's translation pipeline
+ * which is responsible for uploading strings to Crowdin.
+ */
+import fs from "fs";
+import path from "path";
+import ancesdir from "ancesdir";
+import fg from "fast-glob";
+import {extractStrings} from "@khanacademy/wonder-stuff-i18n";
+
+import {getLogger} from "./internal/logger.js";
+
+const rootDir = ancesdir(__dirname);
+
+const logger = getLogger();
+
+export const generateStringsFileForPackage = (pkgName: string) => {
+    const glob = path.join(
+        rootDir,
+        "packages",
+        pkgName,
+        "src",
+        "**",
+        "*.{js,jsx}",
+    );
+    const files = fg.sync(glob);
+
+    if (pkgName === "perseus") {
+        files.push(
+            // NOTE(kevinb): Extracting strings from build files is not ideal since we
+            // lose any I18N comments.  Once Perseus is in its own repo we should move
```

Comment:
> Once Perseus is in its own repo
Today's the day! :) 

Maybe update to a TODO with the Jira ticket to bring `math-input` into this repo?